README.md

```diff
@@ -45,6 +45,9 @@ install that first. Below are instructions for some common operating systems:
 | Termux       | `apt install leveldb`        |
 | FreeBSD      | `pkg install leveldb`        |
 
+> At the time of writing this note, macOS's package installer *brew* requires a different command to install leveldb.  
+`brew install --build-from-source leveldb`
```

Comment:
Please edit the macOS command to add this argument instead. It can be reverted if/when the upstream issue is fixed rather than adding a separate note. Don't worry about alignment in the table for now.